src/plugins/vis_builder/public/visualizations/vislib/area/area_vis_type.ts

```diff
@@ -21,6 +22,7 @@ export const createAreaConfig = (): VisualizationTypeOptions<AreaOptionsDefaults
   title: 'Area',
   icon: 'visArea',
   description: 'Display area chart',
+  stage: VIS_BUILDER_STATE,
```

Comment:
It was from a previous discussion with the experimental flag, do we want to set the stage as 'experimental' for each viz type? Since I see there is a stage field in the VisualizationTypeOptions<T = any> interface, so i set the stage to experimental for all four viz type in vis builder. If not, i think we can get rid of the VIS_BUILDER_STATE = 'experimental' @ashwin-pc 

```
 export interface VisualizationTypeOptions<T = any> {
  readonly name: string;
  readonly title: string;
  readonly description?: string;
  readonly icon: IconType;
  readonly stage?: 'experimental' | 'production';
```